run_tests.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import sys
+
```

Comment:
blank line needed?

Reply:
Hmm - I've not written any tests yet. Perhaps this file is premature...

Reply:
Removed file in d9c1d9b